TicTacToe.Test/BoardTest.cs

```diff
@@ -1,8 +1,18 @@
-﻿using System;
-using Xunit;
+﻿using Xunit;
 
 namespace TicTacToe.Test
 {
+    internal class Helpers
+    {
+        public void FillBoard(Board board, int[] positions, char[] symbols)
```

Comment:
you can make this method static so that you won't have to instantiate the helper class everytime